Sources/SwiftUINavigationCore/Documentation.docc/SwiftUINavigationCore.md

```diff
@@ -1,12 +0,0 @@
-# ``SwiftUINavigationCore``
```

Comment:
Do you know why this file got deleted?

Reply:
Ah, I assumed that I had added all the documentation, my bad.